lld/unittests/EVM/LLDTest.cpp

```diff
@@ -291,6 +290,18 @@ TEST_F(LLDCTest, Assembly) {
       {R_deploy_obj, R_deployed_assemble, A_assembly, A_assembly_deployed},
       {"R_107", "R_107_deployed", "A_38", "A_38.A_38_deployed"});
 
+  // Get linker symbols offsets
+  uint64_t *SymOffsets = nullptr;
+  uint64_t NumOffsets =
+      LLVMSymbolOffsetsEVM(R_assembly, LinkerSymbol[1], &SymOffsets);
+  EXPECT_TRUE(NumOffsets == 3);
+  // These offsets depend on the compiler's optimization level. Since the
```

Comment:
Do they also depend on any changes in the codegen?

Reply:
Right. But anyway I do not expect we will change them often. On the other hand we can drop them, as the don't have big meaning.

Reply:
OK, I'm just trying to avoid fixing these offsets after every such change. If it's applicable, `EXPECT_TRUE(NumOffsets == 3);` should be enough. I use similar logic on FE.